py3-pillow.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: py3-pillow
+  version: 10.0.0
+  epoch: 0
+  description: Python Imaging Library (Fork)
+  copyright:
+    - license: custom:PIL
```

Comment:
We don't have any `- license: custom:PIL`  packages, @kaniini I couldn't find this on https://spdx.org/licenses/, is this the correct format?

Reply:
It appears to be https://spdx.org/licenses/HPND.html